tensorflow/compiler/xla/service/gpu/tests/gpu_noalias_test.cc

```diff
@@ -50,10 +50,18 @@ TEST_F(GpuNoAliasTest, Concat) {
   auto hlo_module = CreateNewVerifiedModule();
   hlo_module->AddEntryComputation(std::move(computation));
 
-  CompileAndVerifyIr(std::move(hlo_module),
-                     R"(CHECK-LABEL: define void @fusion
-                        CHECK-SAME: i8* noalias align {{[0-9]*}} dereferenceable({{[0-9]*}}) %[[OUTPUT_ALLOC:[a-z0-9]*]]
-                        CHECK: %fusion.raw = {{.*}} %[[OUTPUT_ALLOC]])",
+  auto expected_ir = is_built_with_rocm_ ? R"(
+; CHECK-LABEL: define amdgpu_kernel void @fusion
+; CHECK-SAME: i8* noalias align {{[0-9]*}} dereferenceable({{[0-9]*}}) %[[OUTPUT_ALLOC:[a-z0-9]*]]
+; CHECK: %fusion.raw = {{.*}} %[[OUTPUT_ALLOC]]
+)"
+                                         : R"(
+; CHECK-LABEL: define void @fusion
```

Comment:
Can you make it `CHECK-LABEL: define {{.*}} void @fusion` to cover both cases? The cuda/rocm difference is not the point of interest here.

Reply:
Sounds good. `CHECK-LABEL: define{{.*}}void @fusion` so that it is able to match to cuda side of a single space, or ROCm side of `space+amdgpu+space`.